### PR TITLE
Fix some issues with 26-4555

### PR DESCRIPTION
**Pull request by Thrillberg**
## Summary
This PR improves how we submit data to the LGY service endpoint and also fixes the return JSON to the front end so that we can display the confirmation number.

## Related issue(s)
https://app.zenhub.com/workspaces/vagov-product-team-forms-634853151f5c6000165942bc/issues/gh/department-of-veterans-affairs/va.gov-team-forms/702

### CODE REVIEW

File: modules/simple_forms_api/app/models/simple_forms_api/vba_26_4555.rb [hunk 1]
```diff
@@ -12,15 +12,15 @@ def initialize(data)
...
         previousHiApplication: previous_hi_application_payload,
         previousSahApplication: previous_sah_application_payload,
         veteran: veteran_payload,
-        statementOfTruthSignature: data['statement_of_truth_signature'],
-        statementOfTruthCertified: data['statement_of_truth_certified'],
-        formNumber: data['form_number']
+        statementOfTruthSignature: @data['statement_of_truth_signature'],
+        statementOfTruthCertified: @data['statement_of_truth_certified'],
+        formNumber: @data['form_number']
       }
```
Comment (pennja) [inline modules/simple_forms_api/app/models/simple_forms_api/vba_26_4555.rb:24]: Adding `attr_accessor :data` at the top of the file might make this change unnecessary

File: modules/simple_forms_api/app/models/simple_forms_api/vba_26_4555.rb [hunk 2]
```diff
@@ -50,67 +50,77 @@ def track_user_identity; end
...
-        street: address['street'],
-        street2: address['street2'],
-        city: address['city'],
-        state: address['state'],
-        postalCode: address['postal_code']
-      }
+      address = @data.dig('veteran', 'address')
+      if address
+        {
+          isMilitary: true, # address['is_military'],
```
Comment (pennja) [inline modules/simple_forms_api/app/models/simple_forms_api/vba_26_4555.rb:115]: Is this commented out code safe to remove?

File: modules/simple_forms_api/app/models/simple_forms_api/vba_26_4555.rb [hunk 3]
```diff
@@ -54,34 +55,42 @@ def living_situation_payload
...
-          street2: care_facility_address['street2'],
-          city: care_facility_address['city'],
-          state: care_facility_address['state'],
-          postalCode: care_facility_address['postal_code']
+          street: care_facility_address&.fetch('street', nil),
+          street2: care_facility_address&.fetch('street2', nil),
+          city: care_facility_address&.fetch('city', nil),
+          state: care_facility_address&.fetch('state', nil),
+          postalCode: care_facility_address&.fetch('postal_code', nil)
         },
```
Comment (pennja) [inline modules/simple_forms_api/app/models/simple_forms_api/vba_26_4555.rb:63]: Out of curiosity, what is the likelihood of `care_facility_address` or one of its values being nil?

**Review by pennja — APPROVED** [review]
Let's see if my approval means anything (doubtful). Looks good, great work! 👍

**Review by rhasselle-oddball — APPROVED** [review]

**Pull request merged**